action.yml

```diff
@@ -27,6 +27,17 @@ runs:
           using JuliaFormatter
           format(".") ? exit(0) : exit(1)'
       shell: bash
+    - name: Check for formatting errors
+      shell: julia --color=yes --project=@juliaformatter {0}
```

Comment:
Any reason you're starting Julia for this? Something like
```
git diff --name-only && git diff --exit-code
```
should do the same thing (well, apart from the extra line you're printing here).